src/libs/actions/Report.js

```diff
@@ -471,15 +481,36 @@ function subscribeToReportCommentEvents() {
         return;
     }
 
-    Pusher.subscribe(pusherChannelName, 'reportComment', (pushJSON) => {
-        Log.info('[Report] Handled event sent by Pusher', true, {reportID: pushJSON.reportID});
+    // Live-update a report's actions when a 'report comment' event is received.
+    Pusher.subscribe(pusherChannelName, Pusher.TYPE.REPORT_COMMENT, (pushJSON) => {
+        Log.info(
+            `[Report] Handled ${Pusher.TYPE.REPORT_COMMENT} event sent by Pusher`, true, {reportID: pushJSON.reportID},
+        );
         updateReportWithNewAction(pushJSON.reportID, pushJSON.reportAction);
     }, false,
     () => {
         NetworkConnection.triggerReconnectionCallbacks('pusher re-subscribed to private user channel');
     })
         .catch((error) => {
-            Log.info('[Report] Failed to initially subscribe to Pusher channel', true, {error, pusherChannelName});
+            const eventName = Pusher.TYPE.REPORT_COMMENT;
```

Comment:
What's the purpose of setting a constant to a constant here? Can we just put the constant directly in the log?

Reply:
The reason I did this was so I wouldn't have to split the following Log in into 5 separate lines like this:

```js
Log.info(
    '[Report] Failed to subscribe to Pusher channel',
    true,
    {error, pusherChannelName, eventName}
);
```

Instead of 5 lines, my workaround is 2 lines. But if you don't like it, i can definitely change it up :D

Reply:
On second thought, I'll get rid of `const eventName` 👍